Server/Plugins/Debuggers/Debuggers.lua

```diff
@@ -1178,12 +1178,12 @@ function HandleSpideyCmd(a_Split, a_Player)
 	local World = a_Player:GetWorld();
 
 	local Callbacks = {
-		OnNextBlock = function(a_BlockX, a_BlockY, a_BlockZ, a_BlockType, a_BlockMeta)
+		OnNextBlock = function(a_Block, a_BlockType, a_BlockMeta)
```

Comment:
`a_Block` is a poor name - is it the block's position, type, meta, other info, entity? Traditionally, we use `a_BlockPos`.

Reply:
I have changed it.